client/viame-web-common/components/Viewer.vue

```diff
@@ -158,6 +161,9 @@ export default defineComponent({
       populateTypeStyles(meta.customTypeStyling);
       populateConfidenceFilters(meta.confidenceFilters);
       loaded.value = true;
+      fps.value = meta.fps;
+      imageData.value = meta.imageData;
+      videoUrl.value = meta.videoUrl;
```

Comment:
With the web version these aren't available on the meta object.  We need a way that supports both web and desktop.  So I think we need to reimplement the API so that the loadMetaData will include the imageData and videoUrls if needed and take it away from being in the main app section for the web-platform.